app/assets/stylesheets/cafe_works.css.scss

```diff
@@ -0,0 +1,19 @@
+.cafe-date {
+  background: $background_color;
+  border: 2px;
+  border-color: $border_color;
+  border-radius: 10px !important;
+  color: $text_color;
```

Comment:
Name of variable `text_color` should be written in all lowercase letters with hyphens instead of underscores